### DISCUSSION TITLE
New website is now live

### DISCUSSION THREAD

**Discussion by ikeycode** [AerynOS/discussions#5] [Announcements]
Hey folks, our new website is now live, hopefully streamlining a lot of workflows.

Check it out at serpentos.com and the new WIP docs site at serpentos.com/docs! 

Would LOVE any help with improving our packaging & user documentation please! All docsy.dev/hugo/markdown based,
very easy to get started.

**Comment by NomadicCore** [comment]
Looks great and can now try and support in fleshing out the documentation!
